crates/wasi-provider/src/lib.rs

```diff
@@ -71,9 +73,20 @@ impl Provider for WasiProvider {
         // Wrap this in a block so the write lock goes out of scope when we are done
         let mut container_handles = HashMap::new();
         for container in pod.containers() {
+            let mut c = Client::default();
+
+            let image = container
+                .image
+                .clone()
+                .expect("Container must have an image");
+            let image = Reference::try_from(image).unwrap();
+            let store = FileModuleStore::new(&PathBuf::from("/home/rylevick"));
```

Comment:
Should the store be a data member of the `WasiProvider` struct so we don't need to initialize it on every loop?

Reply:
Yes. This should have been marked as WIP.